image/x86_64/macOS_10.12/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM drydock/microbase:{{%TAG%}}
+
+# Prefix all image ENVs with IMAGE_ so there are no confusions.
+ENV IMAGE_REQPROC_DIR /root/reqProc
+ADD . $IMAGE_REQPROC_DIR
+RUN cd $IMAGE_REQPROC_DIR && npm install
+
+ENV IMAGE_EXEC_TEMPLATES_DIR /root/execTemplates
+RUN mkdir -p $IMAGE_EXEC_TEMPLATES_DIR && \
+    wget https://github.com/Shippable/execTemplates/archive/{{%TAG%}}.tar.gz -O /tmp/execTemplates.tar.gz && \
+    tar -xzvf /tmp/execTemplates.tar.gz -C $IMAGE_EXEC_TEMPLATES_DIR --strip-components=1 && \
+    rm /tmp/execTemplates.tar.gz
+
+# With our current design, we need to pull all reqExec binaries.
+# TODO: Pull all binaries that reqExec has here once we start pushing them to S3.
+ENV IMAGE_REQEXEC_DIR /root/reqExec
+RUN mkdir -p $IMAGE_REQEXEC_DIR/x86_64/Ubuntu_16.04 && \
+    wget https://s3.amazonaws.com/shippable-artifacts/reqExec/{{%TAG%}}/reqExec-{{%TAG%}}-x86_64-Ubuntu_16.04.tar.gz -O /tmp/reqExec.tar.gz && \
```

Comment:
Ubuntu binary?

Reply:
Discussed with @Bharath92. This is due to the way we pull reqExec currently -- all of them will get pulled. Since we only have the x86_64 Ubuntu16.04 currently on S3, I've only added that. The rest will be added once we have all artifacts on S3.